app/src/main/java/app/insti/adapter/CommentsAdapter.java

```diff
@@ -106,6 +119,41 @@ public boolean onLongClick(final View v) {
                         @Override
                         public boolean onMenuItemClick(MenuItem item) {
                             switch (item.getItemId()) {
+                                case R.id.edit_comment_option:
+                                    final String temp = textViewComment.getText().toString();
+                                    preEditComments(cardView, textViewComment, editTextComment,send_comment, back_button);
+                                    back_button.setOnClickListener(new View.OnClickListener() {
+                                        @Override
+                                        public void onClick(View v) {
+                                            textViewComment.setText(temp);
+                                            postEditComments(cardView, textViewComment, editTextComment,send_comment, back_button, activity);
+                                        }
+                                    });
+                                    send_comment.setOnClickListener(new View.OnClickListener() {
+                                        @Override
+                                        public void onClick(View v) {
+                                            CommentCreateRequest commentCreateRequest = new CommentCreateRequest(editTextComment.getText().toString());
+                                            retrofitInterface.updateComment("sessionid=" + sessionId, comment.getId(), commentCreateRequest).enqueue(new Callback<Venter.Comment>() {
```

Comment:
There's a clean method in `Utils` to get the `sessionid`. Check any other call.

Reply:
The session id is already present in the getInstance() method . It is not something that has been added now.

Reply:
Yes, but you can instead not use that and use the one from `Utils` instead, since we're trying to transition everything to that. No need to refactor anything else; just use the static function here.